packages/core/src/contexts/auth.tsx

```diff
@@ -40,6 +40,10 @@ export interface AuthContext {
 }
 
 export const AuthContext = createContext<AuthContext>(null as unknown as AuthContext)
+export type LockoutReason = 'logout' | 'timeout'
+
+export const reasonLogout: LockoutReason = 'logout'
+export const reasonTimeout: LockoutReason = 'timeout'
```

Comment:
Might be better as an enum:

```suggestion
export enum LockoutReason {
  Timeout = "Timeout",
  Logout = "Logout",
}
```

And then elsewhere in the code, you'd write:
`lockOutUser(LockoutReason.Timeout)`
or...